.gitignore

```
@@ -62,3 +62,4 @@ python_modules.txt
 quench_data.DAT
 env-fork
 *SIG_TF.json
+!examples/data/*
```

```suggestion
!examples/data/*IN.DAT
```

Otherwise the MFILE and OUT file become untracked when they should not be

---

True, but we have other output files in `examples/data` that we want to be tracked, e.g. the ones we use in `plot_solutions.py`. I could ignore the ones associated with this notebook specifically if you like?

---

I believe you could also use `examples/.gitignore` to make some more specific rules away from the main .gitignore. Excluding all MFiles except the 6 meant to be there. Its not the most elegant situation whichever way we do it.

---

Ok, I've added some specific negations to the `examples/.gitignore`.